Vmc gi version tag (#46589)
hashicorp/terraform-provider-aws: website/docs/r/odb_cloud_vm_cluster.html.markdown

### Basic Usage

#### Resource: `aws_odb_cloud_vm_cluster.with_minimum_parameter`

> 

Reviewer: The `Basic Usage` subheading already applies to this example.

```suggestion
```

    is_incident_logs_enabled      = false
  }
}
```

> #### Resource: `aws_odb_cloud_vm_cluster.with_all_parameters`

Reviewer: ```suggestion
### With Optional Arguments
```

    is_incident_logs_enabled      = true
  }
}

> resource "aws_odb_cloud_vm_cluster" "gi_version_tag_example" {

Reviewer: Typically these three different examples would be split into separate subheadings (e.g. `Basic Usage`, `With All Parameters`, `With GI Version Tag`).

Author: Done

}
```

> #### Resource: `aws_odb_cloud_vm_cluster.gi_version_tag_example`

Reviewer: ```suggestion
### With GI Version Tag
```

  odb_network_id                  = "<aws_odb_network_id>"
  is_local_backup_enabled         = true
  is_sparse_diskgroup_enabled     = true
  license_model                   = "LICENSE_INCLUDED"
  data_storage_size_in_tbs        = 20.0
  db_servers                      = ["my-dbserver-1", "my-db-server-2"]
  db_node_storage_size_in_gbs     = 120.0
  memory_size_in_gbs              = 60
  cluster_name                    = "julia-13"
  timezone                        = "UTC"
  scan_listener_port_tcp          = 1521
  tags = {
    "odb:input_gi_version" = "23.0.0.0"
  }
  data_collection_options {
    is_diagnostics_events_enabled = true
    is_health_monitoring_enabled  = true
    is_incident_logs_enabled      = true
  }
}
```

## Argument Reference

The following arguments are required:

* `cpu_core_count` - (Required) The number of CPU cores to enable on the VM cluster. Changing this will create a new resource.
* `db_servers` - (Required) The list of database servers for the VM cluster. Changing this will create a new resource.
* `display_name` - (Required) A user-friendly name for the VM cluster. Changing this will create a new resource.

> * `gi_version` - (Required) A valid Oracle Grid Infrastructure (GI) software version. To get valid values, use the ListGiVersions operation for the Exadata infrastructure shape. Example: `19.0.0.0`. Changing this creates a new resource. Prefer to provide `odb:input_gi_version` tag. If `odb:input_gi_version` tag is provided, its value must exactly match `gi_version`, otherwise Terraform returns an error. Refer [Resource: `aws_odb_cloud_vm_cluster.gi_version_tag_example`](#resource-aws_odb_cloud_vm_clustergi_version_tag_example).

Reviewer: ```suggestion
* `gi_version` - (Required) A valid Oracle Grid Infrastructure (GI) software version. To get valid values, use the ListGiVersions operation for the Exadata infrastructure shape. Example: `19.0.0.0`. Changing this creates a new resource. Prefer to provide `odb:input_gi_version` tag. If `odb:input_gi_version` tag is provided, its value must exactly match `gi_version`, otherwise Terraform returns an error. See the [`With GI Version Tag`](#with-gi-version-tag) example above.
```

* `cpu_core_count` - (Required) The number of CPU cores to enable on the VM cluster. Changing this will create a new resource.
* `db_servers` - (Required) The list of database servers for the VM cluster. Changing this will create a new resource.
* `display_name` - (Required) A user-friendly name for the VM cluster. Changing this will create a new resource.

> * `gi_version` - (Required) A valid Oracle Grid Infrastructure (GI) software version. To get valid values, use the ListGiVersions operation for the Exadata infrastructure shape. Example: `19.0.0.0`. Changing this creates a new resource. Prefer to provide `odb:input_gi_version` tag. If `odb:input_gi_version` tag is provided, its value must exactly match `gi_version`, otherwise Terraform returns an error. Refer gi_version_tag_example given above.

Reviewer: If the examples are split into distinct subheadings the last sentence can link directly to the GI version tag example instead of referring to the resource instance name.

Author: Done
